src/simperium/http-request.js

```diff
@@ -1,25 +1,28 @@
 // @flow
-import { request } from 'https'
```

Comment:
why this change? You only use request.

Reply:
it's because the tests mock `https`

Reply:
or I thought it was…updating back as it seems to work fine with it